src/command.rs

```diff
@@ -424,34 +425,533 @@ impl <U:WebDriverExtensionRoute> ToJson for WebDriverMessage<U> {
     }
 }
 
+type Capabilities = BTreeMap<String, Json>;
+
 pub trait Parameters: Sized {
     fn from_json(body: &Json) -> WebDriverResult<Self>;
 }
 
+
+/// Trait for objects that can be used to inspect browser capabilities
```

Comment:
Would it be much of an effort to move most of this out a src/capabilities.rs file?